galsim/interpolatedimage.py

```diff
@@ -25,6 +25,7 @@
 from . import _galsim
 from ._galsim import Interpolant
 from ._galsim import Nearest, Linear, Cubic, Quintic, Lanczos, SincInterpolant, Delta
+import numpy as np
```

Comment:
Would this slow down the classes/functions that have to do the importing?  I think I'd rather have the package be slow to initially import than slow down any of the individual functions.  Probably worth benchmarking both options.